iOS/Features/JourneyList/Sources/JourneyList/Journey.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Journey.swift
+//  JourneyList
+//
+//  Created by 이창준 on 11/23/23.
+//
+
+import Foundation
+
+struct Journey: Hashable {
+    let locatoin: String
```

Comment:
location으로 변경 필요할 것 같습니다.
```suggestion
    let location: String
```